plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/helpers/FBShapeHelper.java

```diff
@@ -1,5 +1,6 @@
 /*******************************************************************************
- * Copyright (c) 2023 Martin Erich Jobst
+ * Copyright (c) 2023 Martin Erich Jobst,
```

Comment:
Please change to:
` * Copyright (c) 2023, 2025 Martin Erich Jobst, Primetals Technologies Austria GmbH